state-chain/primitives/src/lib.rs

```diff
@@ -63,6 +64,11 @@ macro_rules! define_wrapper_type {
 			}
 		}
 	};
+
+	($name: ident, $inner: ty) => {
+		$crate::define_wrapper_type!($name, $inner, extra_derives:)
```

Comment:
This needs a trailing `;`.

Alternatively you can just make `extra_derives` optional and remove this second case:

```
	( $name: ident, $inner: ty $(, extra_derives: $(, $extra_derive: ident ),* )? )` 
```
(etc.)